airbyte-integrations/connectors/source-instagram/source_instagram/source.py

```diff
@@ -28,6 +28,14 @@ class Config:
         airbyte_secret=True,
     )
 
+    ig_user_ids: str = Field(
+        None,
+        title="Instagram Business Account ID(s)",
+        description='Comma Separated list of instagram account Ids. Each Instagram account ID must be a 15 digit number.  See the <a href="https://docs.airbyte.com/integrations/sources/instagram">docs</a> for more information.',
+        pattern="^[0-9]{17}(,[0-9]{17})*$",
```

Comment:
Why do you specify 17 digits here? Schouldn't it be 15?

Reply:
No, instagram business account user ids are 17 digit numbers.  https://developers.facebook.com/docs/instagram-api/reference/ig-user/ 
Every example in facebook documentation mentions that Id of IG User is a 17 digit number. I couldn't find the exact documentation that mentions that. However, all the calls I have made to graph api returned a 17 digit number. 

Facebook User Ids, on the other hand, are 15 digit numbers however. But Instagram account Ids are 17 digits.

Reply:
Thanks for resolving this!